src/components/usePreviousValue.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+function usePreviousValue(value) {
```

Comment:
Needs a JS doc

Reply:
also, I recall discussing leaving this out for now. I am guessing you feel differently. But if we want to make this a best practice would you consider these options:

- Discuss somewhere whether we should start using this? Doesn't really feel like we've reached a clear consensus here. FWIW, I find myself on the side of thinking that we don't really need this.
- Contribute some commits to [this FAQ PR I started](https://github.com/Expensify/App/pull/16882) with some guidance so that others know when and how to use this

Reply:
Sounds good, I was mainly doing this in response to another review comment above, but if you feel differently then we don't have to include this. Personally I'm fine either way–I can see why it would be good not to have it though, since we don't want to introduce multiple ways of doing the same thing.